[2018-10-13 10:00:36 AM] - [MainThread] - [output_log.py-line:34] [DEBUG] 连接数据库了
[2018-10-13 10:00:40 AM] - [MainThread] - [connectionpool.py-line:823] [DEBUG] Starting new HTTPS connection (1): api.github.com:443
[2018-10-13 10:00:41 AM] - [MainThread] - [connectionpool.py-line:393] [DEBUG] https://api.github.com:443 "GET /user HTTP/1.1" 200 None
[2018-10-13 10:00:41 AM] - [MainThread] - [output_log.py-line:36] [INFO] {"login": "heyemmby", "id": 32669494, "node_id": "MDQ6VXNlcjMyNjY5NDk0", "avatar_url": "https://avatars0.githubusercontent.com/u/32669494?v=4", "gravatar_id": "", "url": "https://api.github.com/users/heyemmby", "html_url": "https://github.com/heyemmby", "followers_url": "https://api.github.com/users/heyemmby/followers", "following_url": "https://api.github.com/users/heyemmby/following{/other_user}", "gists_url": "https://api.github.com/users/heyemmby/gists{/gist_id}", "starred_url": "https://api.github.com/users/heyemmby/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/heyemmby/subscriptions", "organizations_url": "https://api.github.com/users/heyemmby/orgs", "repos_url": "https://api.github.com/users/heyemmby/repos", "events_url": "https://api.github.com/users/heyemmby/events{/privacy}", "received_events_url": "https://api.github.com/users/heyemmby/received_events", "type": "User", "site_admin": false, "name": "emmby", "company": null, "blog": "", "location": null, "email": null, "hireable": null, "bio": "\u4eba\u4f1a\u98de\u4e48\u3001\uff1f\u5982\u679c\u5168\u4e16\u754c\u7684\u4eba\u90fd\u544a\u8bc9\u6211 \u4e0d\u80fd\u3001\u6211\u662f\u8be5\u653e\u5f03\uff0c\u8fd8\u662f \u9119\u89c6\u4ed6\u4eec\uff0c\uff0c\uff0c", "public_repos": 1, "public_gists": 0, "followers": 0, "following": 0, "created_at": "2017-10-10T09:20:48Z", "updated_at": "2018-10-12T15:18:27Z"}
[2018-10-13 10:00:41 AM] - [MainThread] - [output_log.py-line:36] [INFO] https://api.github.com/user
[2018-10-13 10:00:41 AM] - [MainThread] - [output_log.py-line:34] [DEBUG] 登陆成功
[2018-10-13 10:00:44 AM] - [MainThread] - [output_log.py-line:38] [WARNING] select * from message where uid like '%nima%' or name like '%nima%' or tel like '%nima%'or address like '%nima%';
[2018-10-13 10:00:55 AM] - [MainThread] - [output_log.py-line:38] [WARNING] select * from message where uid like '%cao%' or name like '%cao%' or tel like '%cao%'or address like '%cao%';
[2018-10-13 10:01:14 AM] - [MainThread] - [output_log.py-line:34] [DEBUG] 退出系统，关闭数据库连接
[2018-10-13 10:17:52 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 连接数据库了
[2018-10-13 10:18:31 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 连接数据库了
[2018-10-13 10:21:16 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 连接数据库了
[2018-10-13 10:21:20 AM] - [MainThread] - [connectionpool.py-line:823] [DEBUG] Starting new HTTPS connection (1): api.github.com:443
[2018-10-13 10:21:23 AM] - [MainThread] - [connectionpool.py-line:393] [DEBUG] https://api.github.com:443 "GET /user HTTP/1.1" 200 None
[2018-10-13 10:21:23 AM] - [MainThread] - [output_log.py-line:27] [INFO] {"login": "heyemmby", "id": 32669494, "node_id": "MDQ6VXNlcjMyNjY5NDk0", "avatar_url": "https://avatars0.githubusercontent.com/u/32669494?v=4", "gravatar_id": "", "url": "https://api.github.com/users/heyemmby", "html_url": "https://github.com/heyemmby", "followers_url": "https://api.github.com/users/heyemmby/followers", "following_url": "https://api.github.com/users/heyemmby/following{/other_user}", "gists_url": "https://api.github.com/users/heyemmby/gists{/gist_id}", "starred_url": "https://api.github.com/users/heyemmby/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/heyemmby/subscriptions", "organizations_url": "https://api.github.com/users/heyemmby/orgs", "repos_url": "https://api.github.com/users/heyemmby/repos", "events_url": "https://api.github.com/users/heyemmby/events{/privacy}", "received_events_url": "https://api.github.com/users/heyemmby/received_events", "type": "User", "site_admin": false, "name": "emmby", "company": null, "blog": "", "location": null, "email": null, "hireable": null, "bio": "\u4eba\u4f1a\u98de\u4e48\u3001\uff1f\u5982\u679c\u5168\u4e16\u754c\u7684\u4eba\u90fd\u544a\u8bc9\u6211 \u4e0d\u80fd\u3001\u6211\u662f\u8be5\u653e\u5f03\uff0c\u8fd8\u662f \u9119\u89c6\u4ed6\u4eec\uff0c\uff0c\uff0c", "public_repos": 1, "public_gists": 0, "followers": 0, "following": 0, "created_at": "2017-10-10T09:20:48Z", "updated_at": "2018-10-12T15:18:27Z"}
[2018-10-13 10:21:23 AM] - [MainThread] - [output_log.py-line:27] [INFO] https://api.github.com/user
[2018-10-13 10:21:23 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 登陆成功
[2018-10-13 10:21:28 AM] - [MainThread] - [output_log.py-line:29] [WARNING] select * from message where uid like '%nim%' or name like '%nim%' or tel like '%nim%'or address like '%nim%';
[2018-10-13 10:21:35 AM] - [MainThread] - [output_log.py-line:29] [WARNING] select * from message where uid like '%cao%' or name like '%cao%' or tel like '%cao%'or address like '%cao%';
[2018-10-13 10:24:44 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 连接数据库了
[2018-10-13 10:24:46 AM] - [MainThread] - [connectionpool.py-line:823] [DEBUG] Starting new HTTPS connection (1): api.github.com:443
[2018-10-13 10:24:58 AM] - [MainThread] - [connectionpool.py-line:393] [DEBUG] https://api.github.com:443 "GET /user HTTP/1.1" 200 None
[2018-10-13 10:24:58 AM] - [MainThread] - [output_log.py-line:27] [INFO] {"login": "heyemmby", "id": 32669494, "node_id": "MDQ6VXNlcjMyNjY5NDk0", "avatar_url": "https://avatars0.githubusercontent.com/u/32669494?v=4", "gravatar_id": "", "url": "https://api.github.com/users/heyemmby", "html_url": "https://github.com/heyemmby", "followers_url": "https://api.github.com/users/heyemmby/followers", "following_url": "https://api.github.com/users/heyemmby/following{/other_user}", "gists_url": "https://api.github.com/users/heyemmby/gists{/gist_id}", "starred_url": "https://api.github.com/users/heyemmby/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/heyemmby/subscriptions", "organizations_url": "https://api.github.com/users/heyemmby/orgs", "repos_url": "https://api.github.com/users/heyemmby/repos", "events_url": "https://api.github.com/users/heyemmby/events{/privacy}", "received_events_url": "https://api.github.com/users/heyemmby/received_events", "type": "User", "site_admin": false, "name": "emmby", "company": null, "blog": "", "location": null, "email": null, "hireable": null, "bio": "\u4eba\u4f1a\u98de\u4e48\u3001\uff1f\u5982\u679c\u5168\u4e16\u754c\u7684\u4eba\u90fd\u544a\u8bc9\u6211 \u4e0d\u80fd\u3001\u6211\u662f\u8be5\u653e\u5f03\uff0c\u8fd8\u662f \u9119\u89c6\u4ed6\u4eec\uff0c\uff0c\uff0c", "public_repos": 1, "public_gists": 0, "followers": 0, "following": 0, "created_at": "2017-10-10T09:20:48Z", "updated_at": "2018-10-12T15:18:27Z"}
[2018-10-13 10:24:58 AM] - [MainThread] - [output_log.py-line:27] [INFO] https://api.github.com/user
[2018-10-13 10:24:58 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 登陆成功
[2018-10-13 10:24:59 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 退出系统，关闭数据库连接
[2018-10-13 10:55:46 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 连接数据库了
[2018-10-13 10:55:48 AM] - [MainThread] - [connectionpool.py-line:823] [DEBUG] Starting new HTTPS connection (1): api.github.com:443
[2018-10-13 10:55:51 AM] - [MainThread] - [connectionpool.py-line:393] [DEBUG] https://api.github.com:443 "GET /user HTTP/1.1" 200 None
[2018-10-13 10:55:51 AM] - [MainThread] - [output_log.py-line:27] [INFO] {"login": "heyemmby", "id": 32669494, "node_id": "MDQ6VXNlcjMyNjY5NDk0", "avatar_url": "https://avatars0.githubusercontent.com/u/32669494?v=4", "gravatar_id": "", "url": "https://api.github.com/users/heyemmby", "html_url": "https://github.com/heyemmby", "followers_url": "https://api.github.com/users/heyemmby/followers", "following_url": "https://api.github.com/users/heyemmby/following{/other_user}", "gists_url": "https://api.github.com/users/heyemmby/gists{/gist_id}", "starred_url": "https://api.github.com/users/heyemmby/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/heyemmby/subscriptions", "organizations_url": "https://api.github.com/users/heyemmby/orgs", "repos_url": "https://api.github.com/users/heyemmby/repos", "events_url": "https://api.github.com/users/heyemmby/events{/privacy}", "received_events_url": "https://api.github.com/users/heyemmby/received_events", "type": "User", "site_admin": false, "name": "emmby", "company": null, "blog": "", "location": null, "email": null, "hireable": null, "bio": "\u4eba\u4f1a\u98de\u4e48\u3001\uff1f\u5982\u679c\u5168\u4e16\u754c\u7684\u4eba\u90fd\u544a\u8bc9\u6211 \u4e0d\u80fd\u3001\u6211\u662f\u8be5\u653e\u5f03\uff0c\u8fd8\u662f \u9119\u89c6\u4ed6\u4eec\uff0c\uff0c\uff0c", "public_repos": 1, "public_gists": 0, "followers": 0, "following": 0, "created_at": "2017-10-10T09:20:48Z", "updated_at": "2018-10-12T15:18:27Z"}
[2018-10-13 10:55:51 AM] - [MainThread] - [output_log.py-line:27] [INFO] https://api.github.com/user
[2018-10-13 10:55:51 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 登陆成功
[2018-10-13 10:56:27 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 数据库报错，未插入这个用户信息
[2018-10-13 10:57:52 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 退出系统，关闭数据库连接
[2018-10-13 10:57:53 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 连接数据库了
[2018-10-13 10:57:57 AM] - [MainThread] - [connectionpool.py-line:823] [DEBUG] Starting new HTTPS connection (1): api.github.com:443
[2018-10-13 10:57:59 AM] - [MainThread] - [connectionpool.py-line:393] [DEBUG] https://api.github.com:443 "GET /user HTTP/1.1" 200 None
[2018-10-13 10:57:59 AM] - [MainThread] - [output_log.py-line:27] [INFO] {"login": "heyemmby", "id": 32669494, "node_id": "MDQ6VXNlcjMyNjY5NDk0", "avatar_url": "https://avatars0.githubusercontent.com/u/32669494?v=4", "gravatar_id": "", "url": "https://api.github.com/users/heyemmby", "html_url": "https://github.com/heyemmby", "followers_url": "https://api.github.com/users/heyemmby/followers", "following_url": "https://api.github.com/users/heyemmby/following{/other_user}", "gists_url": "https://api.github.com/users/heyemmby/gists{/gist_id}", "starred_url": "https://api.github.com/users/heyemmby/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/heyemmby/subscriptions", "organizations_url": "https://api.github.com/users/heyemmby/orgs", "repos_url": "https://api.github.com/users/heyemmby/repos", "events_url": "https://api.github.com/users/heyemmby/events{/privacy}", "received_events_url": "https://api.github.com/users/heyemmby/received_events", "type": "User", "site_admin": false, "name": "emmby", "company": null, "blog": "", "location": null, "email": null, "hireable": null, "bio": "\u4eba\u4f1a\u98de\u4e48\u3001\uff1f\u5982\u679c\u5168\u4e16\u754c\u7684\u4eba\u90fd\u544a\u8bc9\u6211 \u4e0d\u80fd\u3001\u6211\u662f\u8be5\u653e\u5f03\uff0c\u8fd8\u662f \u9119\u89c6\u4ed6\u4eec\uff0c\uff0c\uff0c", "public_repos": 1, "public_gists": 0, "followers": 0, "following": 0, "created_at": "2017-10-10T09:20:48Z", "updated_at": "2018-10-12T15:18:27Z"}
[2018-10-13 10:57:59 AM] - [MainThread] - [output_log.py-line:27] [INFO] https://api.github.com/user
[2018-10-13 10:57:59 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 登陆成功
[2018-10-13 10:58:14 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 增加用户信息
[2018-10-13 10:58:14 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] {'name': 'nima', 'age': '99', 'tel': '987654321', 'address': 'ooo'}
[2018-10-13 10:58:30 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 退出系统，关闭数据库连接
[2018-10-13 11:21:57 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 连接数据库了
[2018-10-13 11:22:01 AM] - [MainThread] - [connectionpool.py-line:823] [DEBUG] Starting new HTTPS connection (1): api.github.com:443
[2018-10-13 11:22:03 AM] - [MainThread] - [connectionpool.py-line:393] [DEBUG] https://api.github.com:443 "GET /user HTTP/1.1" 200 None
[2018-10-13 11:22:03 AM] - [MainThread] - [output_log.py-line:27] [INFO] {"login": "heyemmby", "id": 32669494, "node_id": "MDQ6VXNlcjMyNjY5NDk0", "avatar_url": "https://avatars0.githubusercontent.com/u/32669494?v=4", "gravatar_id": "", "url": "https://api.github.com/users/heyemmby", "html_url": "https://github.com/heyemmby", "followers_url": "https://api.github.com/users/heyemmby/followers", "following_url": "https://api.github.com/users/heyemmby/following{/other_user}", "gists_url": "https://api.github.com/users/heyemmby/gists{/gist_id}", "starred_url": "https://api.github.com/users/heyemmby/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/heyemmby/subscriptions", "organizations_url": "https://api.github.com/users/heyemmby/orgs", "repos_url": "https://api.github.com/users/heyemmby/repos", "events_url": "https://api.github.com/users/heyemmby/events{/privacy}", "received_events_url": "https://api.github.com/users/heyemmby/received_events", "type": "User", "site_admin": false, "name": "emmby", "company": null, "blog": "", "location": null, "email": null, "hireable": null, "bio": "\u4eba\u4f1a\u98de\u4e48\u3001\uff1f\u5982\u679c\u5168\u4e16\u754c\u7684\u4eba\u90fd\u544a\u8bc9\u6211 \u4e0d\u80fd\u3001\u6211\u662f\u8be5\u653e\u5f03\uff0c\u8fd8\u662f \u9119\u89c6\u4ed6\u4eec\uff0c\uff0c\uff0c", "public_repos": 1, "public_gists": 0, "followers": 0, "following": 0, "created_at": "2017-10-10T09:20:48Z", "updated_at": "2018-10-12T15:18:27Z"}
[2018-10-13 11:22:03 AM] - [MainThread] - [output_log.py-line:27] [INFO] https://api.github.com/user
[2018-10-13 11:22:03 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 登陆成功
[2018-10-13 11:22:15 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 增加用户信息
[2018-10-13 11:22:15 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] {'name': 'haha', 'age': '23', 'tel': '987654321', 'address': 'dad'}
[2018-10-13 11:22:25 AM] - [MainThread] - [output_log.py-line:29] [WARNING] select * from message where uid like '%ha%' or name like '%ha%' or tel like '%ha%'or address like '%ha%';
[2018-10-13 11:23:46 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 连接数据库了
[2018-10-13 11:23:49 AM] - [MainThread] - [connectionpool.py-line:823] [DEBUG] Starting new HTTPS connection (1): api.github.com:443
[2018-10-13 11:23:51 AM] - [MainThread] - [connectionpool.py-line:393] [DEBUG] https://api.github.com:443 "GET /user HTTP/1.1" 200 None
[2018-10-13 11:23:51 AM] - [MainThread] - [output_log.py-line:27] [INFO] {"login": "heyemmby", "id": 32669494, "node_id": "MDQ6VXNlcjMyNjY5NDk0", "avatar_url": "https://avatars0.githubusercontent.com/u/32669494?v=4", "gravatar_id": "", "url": "https://api.github.com/users/heyemmby", "html_url": "https://github.com/heyemmby", "followers_url": "https://api.github.com/users/heyemmby/followers", "following_url": "https://api.github.com/users/heyemmby/following{/other_user}", "gists_url": "https://api.github.com/users/heyemmby/gists{/gist_id}", "starred_url": "https://api.github.com/users/heyemmby/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/heyemmby/subscriptions", "organizations_url": "https://api.github.com/users/heyemmby/orgs", "repos_url": "https://api.github.com/users/heyemmby/repos", "events_url": "https://api.github.com/users/heyemmby/events{/privacy}", "received_events_url": "https://api.github.com/users/heyemmby/received_events", "type": "User", "site_admin": false, "name": "emmby", "company": null, "blog": "", "location": null, "email": null, "hireable": null, "bio": "\u4eba\u4f1a\u98de\u4e48\u3001\uff1f\u5982\u679c\u5168\u4e16\u754c\u7684\u4eba\u90fd\u544a\u8bc9\u6211 \u4e0d\u80fd\u3001\u6211\u662f\u8be5\u653e\u5f03\uff0c\u8fd8\u662f \u9119\u89c6\u4ed6\u4eec\uff0c\uff0c\uff0c", "public_repos": 1, "public_gists": 0, "followers": 0, "following": 0, "created_at": "2017-10-10T09:20:48Z", "updated_at": "2018-10-12T15:18:27Z"}
[2018-10-13 11:23:51 AM] - [MainThread] - [output_log.py-line:27] [INFO] https://api.github.com/user
[2018-10-13 11:23:51 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 登陆成功
[2018-10-13 11:24:07 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 更新用户信息
[2018-10-13 11:24:07 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] update message set name='yutong', age='25', tel='1880986543', address='hz', update_time='1539401046' where uid='8';
[2018-10-13 11:24:07 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] {'name': 'yutong', 'age': '25', 'tel': '1880986543', 'address': 'hz'}
[2018-10-13 11:24:21 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] (8, 'yutong', '25', '1880986543', 'hz', datetime.datetime(2018, 10, 12, 21, 24, 53), 1539350691, datetime.datetime(2018, 10, 13, 11, 24, 7), 1539401046)
[2018-10-13 11:24:21 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 准备删除这个id的信息
[2018-10-13 11:24:23 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 删除用户信息成功
[2018-10-13 11:24:28 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 退出系统，关闭数据库连接
[2018-10-13 11:29:02 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 连接数据库了
[2018-10-13 11:29:05 AM] - [MainThread] - [connectionpool.py-line:823] [DEBUG] Starting new HTTPS connection (1): api.github.com:443
[2018-10-13 11:29:16 AM] - [MainThread] - [connectionpool.py-line:393] [DEBUG] https://api.github.com:443 "GET /user HTTP/1.1" 200 None
[2018-10-13 11:29:16 AM] - [MainThread] - [output_log.py-line:27] [INFO] {"login": "heyemmby", "id": 32669494, "node_id": "MDQ6VXNlcjMyNjY5NDk0", "avatar_url": "https://avatars0.githubusercontent.com/u/32669494?v=4", "gravatar_id": "", "url": "https://api.github.com/users/heyemmby", "html_url": "https://github.com/heyemmby", "followers_url": "https://api.github.com/users/heyemmby/followers", "following_url": "https://api.github.com/users/heyemmby/following{/other_user}", "gists_url": "https://api.github.com/users/heyemmby/gists{/gist_id}", "starred_url": "https://api.github.com/users/heyemmby/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/heyemmby/subscriptions", "organizations_url": "https://api.github.com/users/heyemmby/orgs", "repos_url": "https://api.github.com/users/heyemmby/repos", "events_url": "https://api.github.com/users/heyemmby/events{/privacy}", "received_events_url": "https://api.github.com/users/heyemmby/received_events", "type": "User", "site_admin": false, "name": "emmby", "company": null, "blog": "", "location": null, "email": null, "hireable": null, "bio": "\u4eba\u4f1a\u98de\u4e48\u3001\uff1f\u5982\u679c\u5168\u4e16\u754c\u7684\u4eba\u90fd\u544a\u8bc9\u6211 \u4e0d\u80fd\u3001\u6211\u662f\u8be5\u653e\u5f03\uff0c\u8fd8\u662f \u9119\u89c6\u4ed6\u4eec\uff0c\uff0c\uff0c", "public_repos": 1, "public_gists": 0, "followers": 0, "following": 0, "created_at": "2017-10-10T09:20:48Z", "updated_at": "2018-10-12T15:18:27Z"}
[2018-10-13 11:29:16 AM] - [MainThread] - [output_log.py-line:27] [INFO] https://api.github.com/user
[2018-10-13 11:29:16 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 登陆成功
[2018-10-13 11:29:19 AM] - [MainThread] - [output_log.py-line:29] [WARNING] select * from message where uid like '%cao%' or name like '%cao%' or tel like '%cao%'or address like '%cao%';
[2018-10-13 11:29:26 AM] - [MainThread] - [output_log.py-line:29] [WARNING] 成功导出csv 文件, new.csv
[2018-10-13 11:33:07 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 退出系统，关闭数据库连接
[2018-10-13 11:33:08 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 连接数据库了
[2018-10-13 11:33:12 AM] - [MainThread] - [connectionpool.py-line:823] [DEBUG] Starting new HTTPS connection (1): api.github.com:443
[2018-10-13 11:33:14 AM] - [MainThread] - [connectionpool.py-line:393] [DEBUG] https://api.github.com:443 "GET /user HTTP/1.1" 200 None
[2018-10-13 11:33:14 AM] - [MainThread] - [output_log.py-line:27] [INFO] {"login": "heyemmby", "id": 32669494, "node_id": "MDQ6VXNlcjMyNjY5NDk0", "avatar_url": "https://avatars0.githubusercontent.com/u/32669494?v=4", "gravatar_id": "", "url": "https://api.github.com/users/heyemmby", "html_url": "https://github.com/heyemmby", "followers_url": "https://api.github.com/users/heyemmby/followers", "following_url": "https://api.github.com/users/heyemmby/following{/other_user}", "gists_url": "https://api.github.com/users/heyemmby/gists{/gist_id}", "starred_url": "https://api.github.com/users/heyemmby/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/heyemmby/subscriptions", "organizations_url": "https://api.github.com/users/heyemmby/orgs", "repos_url": "https://api.github.com/users/heyemmby/repos", "events_url": "https://api.github.com/users/heyemmby/events{/privacy}", "received_events_url": "https://api.github.com/users/heyemmby/received_events", "type": "User", "site_admin": false, "name": "emmby", "company": null, "blog": "", "location": null, "email": null, "hireable": null, "bio": "\u4eba\u4f1a\u98de\u4e48\u3001\uff1f\u5982\u679c\u5168\u4e16\u754c\u7684\u4eba\u90fd\u544a\u8bc9\u6211 \u4e0d\u80fd\u3001\u6211\u662f\u8be5\u653e\u5f03\uff0c\u8fd8\u662f \u9119\u89c6\u4ed6\u4eec\uff0c\uff0c\uff0c", "public_repos": 1, "public_gists": 0, "followers": 0, "following": 0, "created_at": "2017-10-10T09:20:48Z", "updated_at": "2018-10-12T15:18:27Z"}
[2018-10-13 11:33:14 AM] - [MainThread] - [output_log.py-line:27] [INFO] https://api.github.com/user
[2018-10-13 11:33:14 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 登陆成功
[2018-10-13 11:33:19 AM] - [MainThread] - [output_log.py-line:29] [WARNING] select * from message where uid like '%cao%' or name like '%cao%' or tel like '%cao%'or address like '%cao%';
[2018-10-13 11:33:23 AM] - [MainThread] - [output_log.py-line:29] [WARNING] 成功导出html 文件, new.html
[2018-10-13 11:33:23 AM] - [MainThread] - [output_log.py-line:29] [WARNING] 成功导出html 文件, new.html
[2018-10-13 11:33:32 AM] - [MainThread] - [output_log.py-line:29] [WARNING] select * from message where uid like '%ni%' or name like '%ni%' or tel like '%ni%'or address like '%ni%';
[2018-10-13 11:33:48 AM] - [MainThread] - [output_log.py-line:29] [WARNING] select * from message where uid like '%ni%' or name like '%ni%' or tel like '%ni%'or address like '%ni%';
[2018-10-13 11:36:28 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 退出系统，关闭数据库连接
[2018-10-13 11:37:26 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 连接数据库了
[2018-10-13 11:37:28 AM] - [MainThread] - [connectionpool.py-line:823] [DEBUG] Starting new HTTPS connection (1): api.github.com:443
[2018-10-13 11:37:30 AM] - [MainThread] - [connectionpool.py-line:393] [DEBUG] https://api.github.com:443 "GET /user HTTP/1.1" 401 83
[2018-10-13 11:37:30 AM] - [MainThread] - [output_log.py-line:27] [INFO] {"message": "Bad credentials", "documentation_url": "https://developer.github.com/v3"}
[2018-10-13 11:37:30 AM] - [MainThread] - [output_log.py-line:27] [INFO] https://api.github.com/user
[2018-10-13 11:37:30 AM] - [MainThread] - [output_log.py-line:29] [WARNING] 用户信息错误，登陆失败
[2018-10-13 11:37:32 AM] - [MainThread] - [connectionpool.py-line:823] [DEBUG] Starting new HTTPS connection (1): api.github.com:443
[2018-10-13 11:37:33 AM] - [MainThread] - [connectionpool.py-line:393] [DEBUG] https://api.github.com:443 "GET /user HTTP/1.1" 401 83
[2018-10-13 11:37:33 AM] - [MainThread] - [output_log.py-line:27] [INFO] {"message": "Bad credentials", "documentation_url": "https://developer.github.com/v3"}
[2018-10-13 11:37:33 AM] - [MainThread] - [output_log.py-line:27] [INFO] https://api.github.com/user
[2018-10-13 11:37:33 AM] - [MainThread] - [output_log.py-line:29] [WARNING] 用户信息错误，登陆失败
[2018-10-13 11:37:34 AM] - [MainThread] - [connectionpool.py-line:823] [DEBUG] Starting new HTTPS connection (1): api.github.com:443
[2018-10-13 11:37:35 AM] - [MainThread] - [connectionpool.py-line:393] [DEBUG] https://api.github.com:443 "GET /user HTTP/1.1" 401 83
[2018-10-13 11:37:35 AM] - [MainThread] - [output_log.py-line:27] [INFO] {"message": "Bad credentials", "documentation_url": "https://developer.github.com/v3"}
[2018-10-13 11:37:35 AM] - [MainThread] - [output_log.py-line:27] [INFO] https://api.github.com/user
[2018-10-13 11:37:35 AM] - [MainThread] - [output_log.py-line:29] [WARNING] 用户信息错误，登陆失败
[2018-10-13 11:37:35 AM] - [MainThread] - [output_log.py-line:29] [WARNING] 用户信息错误，登陆失败，已锁定
[2018-10-13 11:37:36 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 连接数据库了
[2018-10-13 11:37:36 AM] - [MainThread] - [output_log.py-line:29] [WARNING] 用户信息错误，登陆失败，已锁定
[2018-10-13 11:41:19 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 连接数据库了
[2018-10-13 11:41:22 AM] - [MainThread] - [connectionpool.py-line:823] [DEBUG] Starting new HTTPS connection (1): api.github.com:443
[2018-10-13 11:41:24 AM] - [MainThread] - [connectionpool.py-line:393] [DEBUG] https://api.github.com:443 "GET /user HTTP/1.1" 200 None
[2018-10-13 11:41:24 AM] - [MainThread] - [output_log.py-line:27] [INFO] {"login": "heyemmby", "id": 32669494, "node_id": "MDQ6VXNlcjMyNjY5NDk0", "avatar_url": "https://avatars0.githubusercontent.com/u/32669494?v=4", "gravatar_id": "", "url": "https://api.github.com/users/heyemmby", "html_url": "https://github.com/heyemmby", "followers_url": "https://api.github.com/users/heyemmby/followers", "following_url": "https://api.github.com/users/heyemmby/following{/other_user}", "gists_url": "https://api.github.com/users/heyemmby/gists{/gist_id}", "starred_url": "https://api.github.com/users/heyemmby/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/heyemmby/subscriptions", "organizations_url": "https://api.github.com/users/heyemmby/orgs", "repos_url": "https://api.github.com/users/heyemmby/repos", "events_url": "https://api.github.com/users/heyemmby/events{/privacy}", "received_events_url": "https://api.github.com/users/heyemmby/received_events", "type": "User", "site_admin": false, "name": "emmby", "company": null, "blog": "", "location": null, "email": null, "hireable": null, "bio": "\u4eba\u4f1a\u98de\u4e48\u3001\uff1f\u5982\u679c\u5168\u4e16\u754c\u7684\u4eba\u90fd\u544a\u8bc9\u6211 \u4e0d\u80fd\u3001\u6211\u662f\u8be5\u653e\u5f03\uff0c\u8fd8\u662f \u9119\u89c6\u4ed6\u4eec\uff0c\uff0c\uff0c", "public_repos": 1, "public_gists": 0, "followers": 0, "following": 0, "created_at": "2017-10-10T09:20:48Z", "updated_at": "2018-10-12T15:18:27Z"}
[2018-10-13 11:41:24 AM] - [MainThread] - [output_log.py-line:27] [INFO] https://api.github.com/user
[2018-10-13 11:41:24 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 登陆成功
[2018-10-13 11:45:32 AM] - [MainThread] - [output_log.py-line:29] [WARNING] 增加用户信息错误
[2018-10-13 11:49:27 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 连接数据库了
[2018-10-13 11:49:31 AM] - [MainThread] - [connectionpool.py-line:823] [DEBUG] Starting new HTTPS connection (1): api.github.com:443
[2018-10-13 11:49:32 AM] - [MainThread] - [connectionpool.py-line:393] [DEBUG] https://api.github.com:443 "GET /user HTTP/1.1" 401 83
[2018-10-13 11:49:32 AM] - [MainThread] - [output_log.py-line:27] [INFO] {"message": "Bad credentials", "documentation_url": "https://developer.github.com/v3"}
[2018-10-13 11:49:32 AM] - [MainThread] - [output_log.py-line:27] [INFO] https://api.github.com/user
[2018-10-13 11:49:32 AM] - [MainThread] - [output_log.py-line:29] [WARNING] 用户信息错误，登陆失败
[2018-10-13 11:49:38 AM] - [MainThread] - [connectionpool.py-line:823] [DEBUG] Starting new HTTPS connection (1): api.github.com:443
[2018-10-13 11:49:49 AM] - [MainThread] - [connectionpool.py-line:393] [DEBUG] https://api.github.com:443 "GET /user HTTP/1.1" 200 None
[2018-10-13 11:49:49 AM] - [MainThread] - [output_log.py-line:27] [INFO] {"login": "heyemmby", "id": 32669494, "node_id": "MDQ6VXNlcjMyNjY5NDk0", "avatar_url": "https://avatars0.githubusercontent.com/u/32669494?v=4", "gravatar_id": "", "url": "https://api.github.com/users/heyemmby", "html_url": "https://github.com/heyemmby", "followers_url": "https://api.github.com/users/heyemmby/followers", "following_url": "https://api.github.com/users/heyemmby/following{/other_user}", "gists_url": "https://api.github.com/users/heyemmby/gists{/gist_id}", "starred_url": "https://api.github.com/users/heyemmby/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/heyemmby/subscriptions", "organizations_url": "https://api.github.com/users/heyemmby/orgs", "repos_url": "https://api.github.com/users/heyemmby/repos", "events_url": "https://api.github.com/users/heyemmby/events{/privacy}", "received_events_url": "https://api.github.com/users/heyemmby/received_events", "type": "User", "site_admin": false, "name": "emmby", "company": null, "blog": "", "location": null, "email": null, "hireable": null, "bio": "\u4eba\u4f1a\u98de\u4e48\u3001\uff1f\u5982\u679c\u5168\u4e16\u754c\u7684\u4eba\u90fd\u544a\u8bc9\u6211 \u4e0d\u80fd\u3001\u6211\u662f\u8be5\u653e\u5f03\uff0c\u8fd8\u662f \u9119\u89c6\u4ed6\u4eec\uff0c\uff0c\uff0c", "public_repos": 1, "public_gists": 0, "followers": 0, "following": 0, "created_at": "2017-10-10T09:20:48Z", "updated_at": "2018-10-12T15:18:27Z"}
[2018-10-13 11:49:49 AM] - [MainThread] - [output_log.py-line:27] [INFO] https://api.github.com/user
[2018-10-13 11:49:49 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 登陆成功
[2018-10-13 11:50:02 AM] - [MainThread] - [output_log.py-line:29] [WARNING] 增加用户信息错误
[2018-10-13 11:50:49 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 连接数据库了
[2018-10-13 11:50:53 AM] - [MainThread] - [connectionpool.py-line:823] [DEBUG] Starting new HTTPS connection (1): api.github.com:443
[2018-10-13 11:50:54 AM] - [MainThread] - [connectionpool.py-line:393] [DEBUG] https://api.github.com:443 "GET /user HTTP/1.1" 200 None
[2018-10-13 11:50:54 AM] - [MainThread] - [output_log.py-line:27] [INFO] {"login": "heyemmby", "id": 32669494, "node_id": "MDQ6VXNlcjMyNjY5NDk0", "avatar_url": "https://avatars0.githubusercontent.com/u/32669494?v=4", "gravatar_id": "", "url": "https://api.github.com/users/heyemmby", "html_url": "https://github.com/heyemmby", "followers_url": "https://api.github.com/users/heyemmby/followers", "following_url": "https://api.github.com/users/heyemmby/following{/other_user}", "gists_url": "https://api.github.com/users/heyemmby/gists{/gist_id}", "starred_url": "https://api.github.com/users/heyemmby/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/heyemmby/subscriptions", "organizations_url": "https://api.github.com/users/heyemmby/orgs", "repos_url": "https://api.github.com/users/heyemmby/repos", "events_url": "https://api.github.com/users/heyemmby/events{/privacy}", "received_events_url": "https://api.github.com/users/heyemmby/received_events", "type": "User", "site_admin": false, "name": "emmby", "company": null, "blog": "", "location": null, "email": null, "hireable": null, "bio": "\u4eba\u4f1a\u98de\u4e48\u3001\uff1f\u5982\u679c\u5168\u4e16\u754c\u7684\u4eba\u90fd\u544a\u8bc9\u6211 \u4e0d\u80fd\u3001\u6211\u662f\u8be5\u653e\u5f03\uff0c\u8fd8\u662f \u9119\u89c6\u4ed6\u4eec\uff0c\uff0c\uff0c", "public_repos": 1, "public_gists": 0, "followers": 0, "following": 0, "created_at": "2017-10-10T09:20:48Z", "updated_at": "2018-10-12T15:18:27Z"}
[2018-10-13 11:50:54 AM] - [MainThread] - [output_log.py-line:27] [INFO] https://api.github.com/user
[2018-10-13 11:50:54 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 登陆成功
[2018-10-13 11:50:59 AM] - [MainThread] - [output_log.py-line:29] [WARNING] select * from message where uid like '%yu%' or name like '%yu%' or tel like '%yu%'or address like '%yu%';
[2018-10-13 11:51:06 AM] - [MainThread] - [output_log.py-line:29] [WARNING] select * from message where uid like '%a%' or name like '%a%' or tel like '%a%'or address like '%a%';
[2018-10-13 11:52:42 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 退出系统，关闭数据库连接
[2018-10-13 11:54:42 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 连接数据库了
[2018-10-13 11:54:44 AM] - [MainThread] - [connectionpool.py-line:823] [DEBUG] Starting new HTTPS connection (1): api.github.com:443
[2018-10-13 11:54:45 AM] - [MainThread] - [connectionpool.py-line:393] [DEBUG] https://api.github.com:443 "GET /user HTTP/1.1" 401 83
[2018-10-13 11:54:46 AM] - [MainThread] - [output_log.py-line:27] [INFO] {"message": "Bad credentials", "documentation_url": "https://developer.github.com/v3"}
[2018-10-13 11:54:46 AM] - [MainThread] - [output_log.py-line:27] [INFO] https://api.github.com/user
[2018-10-13 11:54:46 AM] - [MainThread] - [output_log.py-line:29] [WARNING] 用户信息错误，登陆失败
[2018-10-13 11:54:46 AM] - [MainThread] - [connectionpool.py-line:823] [DEBUG] Starting new HTTPS connection (1): api.github.com:443
[2018-10-13 11:54:48 AM] - [MainThread] - [connectionpool.py-line:393] [DEBUG] https://api.github.com:443 "GET /user HTTP/1.1" 401 83
[2018-10-13 11:54:48 AM] - [MainThread] - [output_log.py-line:27] [INFO] {"message": "Bad credentials", "documentation_url": "https://developer.github.com/v3"}
[2018-10-13 11:54:48 AM] - [MainThread] - [output_log.py-line:27] [INFO] https://api.github.com/user
[2018-10-13 11:54:48 AM] - [MainThread] - [output_log.py-line:29] [WARNING] 用户信息错误，登陆失败
[2018-10-13 11:54:49 AM] - [MainThread] - [connectionpool.py-line:823] [DEBUG] Starting new HTTPS connection (1): api.github.com:443
[2018-10-13 11:54:50 AM] - [MainThread] - [connectionpool.py-line:393] [DEBUG] https://api.github.com:443 "GET /user HTTP/1.1" 401 83
[2018-10-13 11:54:50 AM] - [MainThread] - [output_log.py-line:27] [INFO] {"message": "Bad credentials", "documentation_url": "https://developer.github.com/v3"}
[2018-10-13 11:54:50 AM] - [MainThread] - [output_log.py-line:27] [INFO] https://api.github.com/user
[2018-10-13 11:54:50 AM] - [MainThread] - [output_log.py-line:29] [WARNING] 用户信息错误，登陆失败
[2018-10-13 11:54:50 AM] - [MainThread] - [output_log.py-line:29] [WARNING] 用户信息错误，登陆失败，已锁定
[2018-10-13 11:54:54 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 连接数据库了
[2018-10-13 11:54:54 AM] - [MainThread] - [output_log.py-line:29] [WARNING] 用户信息错误，登陆失败，已锁定
[2018-10-13 11:56:02 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 连接数据库了
[2018-10-13 11:56:34 AM] - [MainThread] - [connectionpool.py-line:823] [DEBUG] Starting new HTTPS connection (1): api.github.com:443
[2018-10-13 11:56:35 AM] - [MainThread] - [connectionpool.py-line:393] [DEBUG] https://api.github.com:443 "GET /user HTTP/1.1" 401 83
[2018-10-13 11:56:35 AM] - [MainThread] - [output_log.py-line:27] [INFO] {"message": "Bad credentials", "documentation_url": "https://developer.github.com/v3"}
[2018-10-13 11:56:35 AM] - [MainThread] - [output_log.py-line:27] [INFO] https://api.github.com/user
[2018-10-13 11:56:35 AM] - [MainThread] - [output_log.py-line:29] [WARNING] 用户信息错误，登陆失败
[2018-10-13 11:56:39 AM] - [MainThread] - [connectionpool.py-line:823] [DEBUG] Starting new HTTPS connection (1): api.github.com:443
[2018-10-13 11:56:40 AM] - [MainThread] - [connectionpool.py-line:393] [DEBUG] https://api.github.com:443 "GET /user HTTP/1.1" 200 None
[2018-10-13 11:56:40 AM] - [MainThread] - [output_log.py-line:27] [INFO] {"login": "heyemmby", "id": 32669494, "node_id": "MDQ6VXNlcjMyNjY5NDk0", "avatar_url": "https://avatars0.githubusercontent.com/u/32669494?v=4", "gravatar_id": "", "url": "https://api.github.com/users/heyemmby", "html_url": "https://github.com/heyemmby", "followers_url": "https://api.github.com/users/heyemmby/followers", "following_url": "https://api.github.com/users/heyemmby/following{/other_user}", "gists_url": "https://api.github.com/users/heyemmby/gists{/gist_id}", "starred_url": "https://api.github.com/users/heyemmby/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/heyemmby/subscriptions", "organizations_url": "https://api.github.com/users/heyemmby/orgs", "repos_url": "https://api.github.com/users/heyemmby/repos", "events_url": "https://api.github.com/users/heyemmby/events{/privacy}", "received_events_url": "https://api.github.com/users/heyemmby/received_events", "type": "User", "site_admin": false, "name": "emmby", "company": null, "blog": "", "location": null, "email": null, "hireable": null, "bio": "\u4eba\u4f1a\u98de\u4e48\u3001\uff1f\u5982\u679c\u5168\u4e16\u754c\u7684\u4eba\u90fd\u544a\u8bc9\u6211 \u4e0d\u80fd\u3001\u6211\u662f\u8be5\u653e\u5f03\uff0c\u8fd8\u662f \u9119\u89c6\u4ed6\u4eec\uff0c\uff0c\uff0c", "public_repos": 1, "public_gists": 0, "followers": 0, "following": 0, "created_at": "2017-10-10T09:20:48Z", "updated_at": "2018-10-12T15:18:27Z"}
[2018-10-13 11:56:40 AM] - [MainThread] - [output_log.py-line:27] [INFO] https://api.github.com/user
[2018-10-13 11:56:40 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 登陆成功
[2018-10-13 11:56:59 AM] - [MainThread] - [output_log.py-line:29] [WARNING] 增加用户信息错误
[2018-10-13 11:57:24 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 增加用户信息
[2018-10-13 11:57:24 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] {'name': 'bingo', 'age': '25', 'tel': '0987654321', 'address': 'hz'}
[2018-10-13 11:57:34 AM] - [MainThread] - [output_log.py-line:29] [WARNING] select * from message where uid like '%bingo%' or name like '%bingo%' or tel like '%bingo%'or address like '%bingo%';
[2018-10-13 11:57:44 AM] - [MainThread] - [output_log.py-line:29] [WARNING] 成功导出csv 文件, new.csv
[2018-10-13 11:57:52 AM] - [MainThread] - [output_log.py-line:29] [WARNING] select * from message where uid like '%a%' or name like '%a%' or tel like '%a%'or address like '%a%';
[2018-10-13 11:57:59 AM] - [MainThread] - [output_log.py-line:29] [WARNING] 成功导出html 文件, new.html
[2018-10-13 11:58:40 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 更新用户信息
[2018-10-13 11:58:40 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] update message set name='yutong', age='25', tel='188987654', address='hz', update_time='1539403115' where uid='10';
[2018-10-13 11:58:40 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] {'name': 'yutong', 'age': '25', 'tel': '188987654', 'address': 'hz'}
[2018-10-13 11:58:59 AM] - [MainThread] - [output_log.py-line:29] [WARNING] select * from message where uid like '%yu%' or name like '%yu%' or tel like '%yu%'or address like '%yu%';
[2018-10-13 11:59:17 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] (10, 'yutong', '25', '188987654', 'hz', datetime.datetime(2018, 10, 13, 11, 22, 15), 1539400933, datetime.datetime(2018, 10, 13, 11, 58, 40), 1539403115)
[2018-10-13 11:59:17 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 准备删除这个id的信息
[2018-10-13 11:59:20 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 删除用户信息成功
[2018-10-13 11:59:24 AM] - [MainThread] - [output_log.py-line:25] [DEBUG] 退出系统，关闭数据库连接
